cmd/queue/execprobe_test.go

```diff
@@ -46,9 +47,9 @@ func TestProbeQueueConnectionFailure(t *testing.T) {
 }
 
 func TestProbeQueueNotReady(t *testing.T) {
-	queueProbed := 0
```

Comment:
I wonder where the race is from. Everything here is kind of synchronous, no?

Reply:
turns out no! We hit one in https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_serving/8899/pull-knative-serving-unit-tests/1290529858908065792. Looks like the data race detector was just giving us a false negative. The tldr is the httptest servers run in their own goroutines so _technically_ they're async to the main test - even though in practice the flow is synchronous there's no happens-before as far as go knows.